src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -47,6 +47,7 @@ export const findReplaceIcon = registerIcon('find-replace', Codicon.replace, nls
 export const findReplaceAllIcon = registerIcon('find-replace-all', Codicon.replaceAll, nls.localize('findReplaceAllIcon', 'Icon for \'Replace All\' in the editor find widget.'));
 export const findPreviousMatchIcon = registerIcon('find-previous-match', Codicon.arrowUp, nls.localize('findPreviousMatchIcon', 'Icon for \'Find Previous\' in the editor find widget.'));
 export const findNextMatchIcon = registerIcon('find-next-match', Codicon.arrowDown, nls.localize('findNextMatchIcon', 'Icon for \'Find Next\' in the editor find widget.'));
+export const findAllMatchesIcon = registerIcon('find-all-in-current-file', Codicon.search, nls.localize('findAllMatchesIcon', 'Icon for \'Find All in Current File\' in the editor find widget.'));
```

Comment:
```suggestion
export const findAllMatchesIcon = registerIcon('find-all-in-current-file', Codicon.newFile, nls.localize('findAllMatchesIcon', 'Icon for \'Find All in Current File\' in the editor find widget.'));
```

Reply:
@ssigwart could you merge this suggestion? It's the correct icon, matching the search pane next to the vertical Activity Bar.

Reply:
@zm-cttae, I had taken a look at this when you first suggested, but this branch is so far behind main now and there were major merge conflicts. It's not worth my time fixing that without some type of feedback from Microsoft indicating this PR may be merged.

Reply:
It'd be possible to make the change and resolve the review.. looking to keep my lists clear

Reply:
It doesn't seem that useful to change the icon when the whole thing won't work in its current state due to VS Code internal changes since I originally wrote this. I assume that Microsoft has other priorities and won't be looking at this, but I leave it open just in case.